.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Question
+about: A question which is not a bug report but sth which may need to be addressed.
+title: ''
+labels: Question
+assignees: ''
+
+---
+
+**Version**: **REPLACEME**
```

Comment:
Provide more clue on how to get the version.
```suggestion
#### Version
<!--- Paste output from `cobbler version` -->
```paste below

\```
```

Reply:
Done as desired